packages/neuron-ui/src/widgets/QRScanner/index.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { useState, useEffect, useMemo, useRef, useCallback } from 'react'
+import { Card, Button, InputGroup, FormControl } from 'react-bootstrap'
+import { Scan as ScanIcon } from 'grommet-icons'
+import { useTranslation } from 'react-i18next'
+import jsQR from 'jsqr'
+
+import Dialog from '../Dialog'
+import { drawPolygon } from '../../utils/canvasActions'
+
+interface QRScannerProps {
+  title: string
+  label: string
+  prompt?: string
+  onConfirm: Function
+  styles?: { [index: string]: any }
+}
+
+const QRScanner = ({ title, label, onConfirm, styles }: QRScannerProps) => {
```

Comment:
Just a question: does this work cross-platform?

Reply:
According to the code, yes. [Online Demo](https://cozmo.github.io/jsQR/)